packages/builder/src/components/portal/licensing/licensingBanners.js

```diff
@@ -141,6 +145,36 @@ const buildPaymentFailedBanner = () => {
   }
 }
 
+const buildUsersAboveLimitBanner = EXPIRY_KEY => {
+  const userLicensing = get(licensing)
+  return {
+    key: EXPIRY_KEY,
+    type: BANNER_TYPES.WARNING,
+    criteria: () => {
+      return (
```

Comment:
Important logic.

Reply:
(i removed the negation but forgot to remove the comment. Will sort.

Reply:
Ah right, I misunderstood when startDate was going to be added. Updated and simplified this!